### PR TITLE
#31 sub-workflow for phage prediction

### LINKED ISSUE #31: Sub-workflow for Phage prediction
### Description of feature

Create a sub-workflow for AMR detection with the following modules from nf-core.
- [ ] [CheckV](https://github.com/nf-core/modules/tree/master/modules/nf-core/checkv)

The following modules require writing nextflow modules before integration
- [ ] [Virsorter2](https://github.com/jiarong/VirSorter2)

### PR DESCRIPTION
**Pull request by jimmyliu1326**
- Created a subworkflow for phage prediction
- Imports local Virsorter2 module
- Closes #31

### CODE REVIEW

File: subworkflows/local/phage.nf [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// INCLUDE MODULES
+include { VIRSORTER2 } from '../../modules/local/virsorter2/main.nf'  
+
```
Comment (anwarMZ) [inline subworkflows/local/phage.nf:3]: include { VIRSORTER2 } from '../../modules/local/virsorter2/config/main.nf'

File: subworkflows/local/phage.nf [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// INCLUDE MODULES
+include { VIRSORTER2 } from '../../modules/local/virsorter2/main.nf'
```
Comment (anwarMZ) [inline subworkflows/local/phage.nf:2]: `include { VIRSORTER2 } from '../../modules/local/virsorter2/run/main.nf`

File: subworkflows/local/phage.nf [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+// Plasmid ID workflow
+workflow PHAGE {
+    take: genome
+    main:
+        
+        // inititalize version channel
+        ch_versions = Channel.empty()
+
+        if (!params.skip_virsorter2) {
```
Comment (anwarMZ) [inline subworkflows/local/phage.nf:12]: sub-workflow needs to be updated with new structure of virsorter2 config module

**Review by anwarMZ — CHANGES_REQUESTED** [review]
PHAGE Sub-workflow needs to be updated with the new module structure

File: subworkflows/local/phage.nf (hunk 3 above)
Comment (jimmyliu1326) [reply subworkflows/local/phage.nf:15]: resolved in [2fa40cd](https://github.com/cidgoh/nf-seqqc/pull/51/commits/2fa40cdfeba908da651b54951524de947c1c5dcb)

File: subworkflows/local/phage.nf (hunk 2 above)
Comment (jimmyliu1326) [reply subworkflows/local/phage.nf]: resolved in [2fa40cd](https://github.com/cidgoh/nf-seqqc/pull/51/commits/2fa40cdfeba908da651b54951524de947c1c5dcb)

**Review by anwarMZ — APPROVED** [review]

**Pull request merged**